python/utils/LinkedLibraryHolder.cpp

```diff
@@ -24,6 +24,11 @@ void __nvqir__setCircuitSimulator(nvqir::CircuitSimulator *);
 }
 
 namespace cudaq {
+
+/// @brief Keep an eye out for requests to ignore
+/// target modification.
+extern bool disallowTargetModification;
```

Comment:
Can't this be the definition? Or does it really need to be in the cudaq.cpp file?

Reply:
libcudaq-pyscf.so cannot link to _pycudaq.so since it is constructed as a linkable module (cmake terms) from the pybind cmake functions. I had to find some go between, and both link to libcudaq.so. Not the greatest solution, I know. Maybe I can file an issue and come back and consider pulling LinkedLibraryHolder out into its own libpy-utils.so package, and have pyscf link to it.